### PR TITLE
Change PSR pull for new Funding Time Allocation

### PR DESCRIPTION
**Pull request by kmagida110**

### CODE REVIEW

File: temporary-psr-generation/psr-generator.r [hunk 1]
```diff
@@ -97,7 +96,7 @@ generate_psr_for_report_id <- function(.id) {
     ) as MostRecentFamilyDetermination
       on MostRecentFamilyDetermination.FamilyId = Child.FamilyId
     where Site.OrganizationId = {.report$OrganizationId}
-      and F.Source = {.report$Type}
+      and F.Source IN (0,1)
```
Comment (jlhogan) [inline temporary-psr-generation/psr-generator.r:99]: 0 is currently the only valid FundingSource enum value

File: temporary-psr-generation/psr-generator.r [hunk 2]
```diff
@@ -126,27 +125,24 @@ generate_psr_for_report_id <- function(.id) {
...
+     Child.C4KFamilyCaseNumber as FamilyId
      from Funding
      inner join Enrollment on Enrollment.Id = EnrollmentId
      inner join Site on Site.Id = SiteId
      inner join Child on Child.Id = ChildId
-     inner join Family F on Child.FamilyId = F.Id
      INNER JOIN C4KCertificate C4KC on Enrollment.ChildId = C4KC.ChildId
-     where Site.OrganizationId = {.report$OrganizationId}
-     and Source = 1 -- C4K
+     where Site.OrganizationId = {.report$OrganizationId} -- C4K is always a secondary source
```
Comment (jlhogan) [inline temporary-psr-generation/psr-generator.r:135]: not clear to me what this comment means. secondary source?

Comment (kmagida110) [reply temporary-psr-generation/psr-generator.r:135]: Supplemental may make more sense, ie. it isn't one of the ones that we structure reports around

File: temporary-psr-generation/psr-generator.r (hunk 1 above)
Comment (kmagida110) [reply temporary-psr-generation/psr-generator.r:99]: yup, missed this

**Pull request merged**